docs/workers.md

```diff
@@ -120,7 +120,8 @@ redis:
 See the sample config for the full documentation of each option.
 
 Under **no circumstances** should the replication listener be exposed to the
-public internet; it has no authentication and is unencrypted.
+public internet; it is unencrypted and unauthenticated unless
+`worker_replication_secret` is configured.
```

Comment:
Hopefully this doesn't make it sound like it is encrypted if `worker_replication_secret` is configured?

Reply:
I'll try, I've rewritten it a few times already and failed to come up with something clearer. 😢

Reply:
I gave rewording it a handful of tries, in case it helps:
```suggestion
public internet; it is unencrypted and unauthenticated unless
`worker_replication_secret` is configured, in which case it is authenticated but
still unencrypted.
```

```suggestion
public internet; it is unencrypted and unauthenticated.
`worker_replication_secret` may be configured to enable authentication, but not
encryption.
```

Reply:
@squahtx I ended up using bullets to separate the two points, what do you think now?

Reply:
👍 it's pretty unambiguous now